sdk/core/azure-core/azure/core/messaging.py

```diff
@@ -0,0 +1,135 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import uuid
+from  base64 import b64decode
+from datetime import tzinfo, timedelta, datetime
+
+try:
+    from datetime import timezone
+    TZ_UTC = timezone.utc  # type: ignore
+except ImportError:
+    class UTC(tzinfo):
```

Comment:
I am thinking if we can leverage/refactor the code https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/azure/core/pipeline/policies/_utils.py#L30